src/main/scala/cromwell/Main.scala

```diff
@@ -68,7 +67,10 @@ object Main extends App {
         case _ => throw new RuntimeException("Expecting a JSON object")
       }
 
-      val workflowManagerSystem = new WorkflowManagerSystem {}
+      lazy val realDataAccess = new RealDataAccess
```

Comment:
I also vote against `Real`, that's a terrible name largely intended to get people to think up a better one.   :smile: 
As for voting *for* a name, I could support `DbDataAccess`.